contributors/design-proposals/apiserver-build-in-admission-plugins.md

```diff
@@ -0,0 +1,31 @@
+# Build some Admission Controllers into the Generic API server library
+
+**Related PR:**
+
+| Topic | Link |
+| ----- | ---- |
+| Admission Control | https://github.com/kubernetes/community/blob/master/contributors/design-proposals/admission_control.md |
+
+## Introduction
+
+At the moment admission controllers are implemented as plugins and they have to be compiled into the final binary in order to be used at a later time. To streamline this process especially for aggregated API servers we would like to build some plugins into the generic API server library.
```

Comment:
I think you should add some detail about the admission chain being a key reason why people would use an aggregated API server.

There should also be some detail about what an admission plugin initializer does.  An admission plugin initializer is used to pass additional configuration and runtime references to admission plugins.  While anyone can author and register one, having a known set of provided references let's people focus on what they need their admission plugin to do instead of paying attention to wiring.